persist/sqlite/migrations.go

```diff
@@ -12,6 +12,12 @@ import (
 	"go.uber.org/zap"
 )
 
+// migrateVersion37 recalculates the contract metrics to remove the pending contract
+// values from the metrics.
+func migrateVersion37(tx *txn, log *zap.Logger) error {
+	return recalcContractMetrics(tx, log)
```

Comment:
I noticed you decode registry read/write in your v2 contracts query (see `recalc.go:171`)

Reply:
💪🏻